swap/src/cli/cancel_and_refund.rs

```diff
@@ -68,15 +87,8 @@ pub async fn cancel(
             }
```

Comment:
Add an else case here if the error code is not RpcVerifyError nor RpcVerifyAlreadyInChain and add a log message that gives some context